backend/LeetWars.BuilderService/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM scratch
+WORKDIR /
```

Comment:
it should not be empty

Reply:
it should build other backend images ?

Reply:
no, all Dockerfiles should have the same structure and build only inself